msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -1521,6 +1521,131 @@ The following code provides an example of meeting end event payload:
 | **value.EndTime** | The meeting end time in UTC. |
 | **locale**| The locale of the message set by the client. |
 
+## Get participant events
+
+Your bot can receive real-time meeting details such as participant join and leave events through the meeting event subscription in Developer Portal. For example, if you want your bot app to notify or return the time stamp when an attendee joined or left the meeting, subscribe to these participant events in Developer Portal.
+
+> [!NOTE]
+>
+> * Participant events is supported only in scheduled private meetings.
+> * For a bot to receive participant events, ensure that you add the bot to the meeting before a participant joins or leaves the meeting.
+
+To subscribe to participant events, follow these steps:
+
+1. In [Developer Portal](https://dev.teams.microsoft.com/) open your bot app or import an existing app.
+1. Ensure that the `OnlineMeetingParticipant.Read.Chat` RSC delegated permissions is configured in your app manifest for scheduled private meetings.
```

Comment:
This section could be moved last in the bullet list, after the screenshot

Reply:
Updated